pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/TopicOperation.java

```diff
@@ -55,4 +55,5 @@ public enum TopicOperation {
 
     SET_REPLICATED_SUBSCRIPTION_STATUS,
     GET_REPLICATED_SUBSCRIPTION_STATUS,
+    TRIM_TOPIC,
```

Comment:
I think we also need to add the new operation to https://github.com/apache/pulsar/blob/efb251cec6ff7dd7a92bc8ff99e8b2fe04d1eb00/pulsar-broker-common/src/main/java/org/apache/pulsar/broker/authorization/PulsarAuthorizationProvider.java#L548-L556